django/urls/resolvers.py

```diff
@@ -76,50 +77,14 @@ def get_ns_resolver(ns_pattern, resolver):
     return RegexURLResolver(r'^/', [ns_resolver])
 
 
-class LocaleRegexDescriptor:
-    def __get__(self, instance, cls=None):
-        """
-        Return a compiled regular expression based on the active language.
-        """
-        if instance is None:
-            return self
-        # As a performance optimization, if the given regex string is a regular
-        # string (not a lazily-translated string proxy), compile it once and
-        # avoid per-language compilation.
-        if isinstance(instance._regex, str):
-            instance.__dict__['regex'] = self._compile(instance._regex)
-            return instance.__dict__['regex']
-        language_code = get_language()
-        if language_code not in instance._regex_dict:
-            instance._regex_dict[language_code] = self._compile(str(instance._regex))
-        return instance._regex_dict[language_code]
-
-    def _compile(self, regex):
-        """
-        Compile and return the given regular expression.
-        """
-        try:
-            return re.compile(regex)
-        except re.error as e:
-            raise ImproperlyConfigured(
-                '"%s" is not a valid regular expression: %s' % (regex, e)
-            )
-
-
-class LocaleRegexProvider:
+class BaseURL:
     """
-    A mixin to provide a default regex property which can vary by active
-    language.
+    Base class for URL instances.
     """
-    def __init__(self, regex):
-        # regex is either a string representing a regular expression, or a
-        # translatable string (using gettext_lazy) representing a regular
-        # expression.
-        self._regex = regex
-        self._regex_dict = {}
+    pass
```

Comment:
I *think* the `pass` is actually unnecessary here, because a doc-string is also a statement.